File: Springboot/src/main/java/com/playarena/security/JwtUtil.java
package com.playarena.security;

import org.springframework.stereotype.Component;
import java.util.Base64;

@Component
public class JwtUtil {

    public String generateToken(String email) {
        return Base64.getEncoder().encodeToString(email.getBytes());
    }

    public String extractEmail(String token) {
        return new String(Base64.getDecoder().decode(token));
    }

    public boolean validateToken(String token) {
        try {
            extractEmail(token);
            return true;
        } catch (Exception e) {
            return false;
        }
    }
}

File: project/config/JwtUtil.php
<?php
class JwtUtil {
    public static function generateToken($email) {
        return base64_encode($email);
    }

    public static function extractEmail($token) {
        if (empty($token)) {
            return null;
        }
        $decoded = base64_decode($token, true);
        if ($decoded === false) {
            return null;
        }
        return $decoded;
    }

    public static function validateToken($token) {
        try {
            $email = self::extractEmail($token);
            return !empty($email);
        } catch (Exception $e) {
            return false;
        }
    }
}
